R2Northstar/NorthstarLauncher
# Remove uses of Autohook from `clientvideooverrides.cpp`

Depends on #780 

### Code review:
- The original function is prefixed with `o_p` so any times where the old AUTOHOOK was calling the original function it should now be calling that instead.
- The hook function is prefixed with `h_` so any times where we would be wanting to call the hooked function from other functions should now be calling that instead

### Testing:
- Check logs to make sure that all of the changed hooks are being created properly
- Make sure that `.bik` files can still be loaded from mod directories (custom main menus)
